Classes/Domain/Model/FeedbackCollection.php

```diff
@@ -75,4 +75,9 @@ public function jsonSerialize()
             'feedbacks' => $feedbacks
         ];
     }
+
+    public function reset()
+    {
+        $this->feedbacks = [];
+    }
```

Comment:
this does not really have anything to do with the core of the change; but we were missing a way to reset feedbacks beforehand.